LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Another unintentional file removal? I don't have strong feelings on having a `LICENSE` file or not.